.github/workflows/main.yml

```diff
@@ -18,6 +18,8 @@ jobs:
             make check
             make tests
             make misalign
+            make clean ENABLE_ARC=1
```

Comment:
Should be `make ENABLE_ARC=1 clean tests`

Reply:
`make ENABLE_ARC=1` clean all for compile ARC version `cache.o`, `make ENABLE_ARC=1` tests for ARC tests.